bb.edn

```diff
@@ -13,9 +13,11 @@
              [babashka.process :as p :refer [process]])
 
   clean {:doc "Start from clean slate."
-         :task (do (run! fs/delete (fs/list-dir (fs/file "resources" "public" "js") "**.*"))
-                   (fs/delete-tree ".cpcache")
-                   (fs/delete-tree ".shadow-cljs"))}
+         :task (let [js-dir (fs/file "resources" "public" "js")]
```

Comment:
I think this run! thing could also just be (fs/delete-tree (fs/path "resources "public "js")`? Not sure why it looks so complicated.

Reply:
Ok, will fix! It looks complicated because I do not know what I am doing.

Reply:
Oh I see you were saying to original is complicated lol. :+1: